src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss

```diff
@@ -0,0 +1,102 @@
+@import '../../../../assets/stylesheets/base';
+
+.typeahead-dropdown {
+  padding-left: 0;
+    
+    .neutral-entry {
+      font-style: italic !important;
+      opacity: 0.5 !important;
+    }
+
+    .details-dropdown {
+      width: $width100;
+    }
+
+    .btn-bar {
+      margin-left: 5px;
+      span {
+        z-index: 11;
+      }
+    }
+
+    .dropdown-search {
+      position: relative;
+      width: 40%;
+      height: 1px;
+      z-index: 10000;
+
+      .list-container {
+        min-width: em(200);
+        z-index: 10;
+        background-color: $color-pf-black-100;
+        border: 1px solid $color-pf-black-500;
+
+        .dropdown-search-icon {
+          position: absolute;
+          line-height: em(30);
+          padding-left: em(5);
+          height: inherit;
+          color: $color-pf-black-500;
+        }
+
+        input {
+          outline: none;
+          width: $width100;
+          height: em(30);
+          border: none;
+          border-bottom: 1px solid $color-pf-black-500;
+          padding-left: em(20);
+        }
+
+        .dropdown-list {
+          max-height: em(150);
+          overflow-y: scroll;
```

Comment:
default overflow for `overflow-x` is visible - will the dropdown list expand to accommodate larger text strings with a max-width?